diff --git a/loop.py b/loop.py
index 4286c67..f909e5e 100644
--- a/loop.py
+++ b/loop.py
@@ -108,7 +108,7 @@ def _extract_message(update):
                                    'inline_query',
                                    'chosen_inline_result',
                                    'shipping_query',
-                                   'pre_checkout_query'])
+                                   'pre_checkout_query','poll','my_chat_member','chat_member'])
     return key, update[key]
 
 def _infer_handler_function(bot, h):
